wandi/easyadmin-plus-bundle/0.3/config/packages/wandi_easyadmin_plus.yaml

```diff
@@ -0,0 +1,12 @@
+#wandi_easy_admin_plus:
```

Comment:
We should avoid having config in comments like this. I would prefer to see a link to the docs instead.